.teamcity/Installers/BuildTypes/SandboxAdmin.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed to the Ed-Fi Alliance under one or more agreements.
+ * The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ * See the LICENSE and NOTICES files in the project root for more information.
+ */
+package installers.buildTypes
```

Comment:
add a blank line please

Reply:
I see that's what has been done elsewhere. I don't have a strong feeling about it - but I am going to change the VS Code settings.json to have an extra line break after the header. I was relying on that file to get the headers right for me.

Reply:
NVM, the editor _is_ inserting a  line break there, and I was pasting over that line.

Reply:
Done